superset/utils/screenshots.py

```diff
@@ -204,41 +219,30 @@ class ChartScreenshot(BaseScreenshot):
     thumbnail_type: str = "chart"
     element: str = "chart-container"
 
-    def __init__(
-        self,
-        url: str,
-        digest: str,
-        window_size: Optional[WindowSize] = None,
-        thumb_size: Optional[WindowSize] = None,
-    ):
-        # Chart reports are in standalone="true" mode
+    def __init__(self, url: str, digest: str, *args: ScreenshotDetails):
```

Comment:
@betodealmeida what do you think of this as a way to resolve the too-many-arguments linting error? I see that we're ignoring it in a lot of places, but I'd like to find a solution that we can use as a pattern for bundling up extra args/kwargs, and wanted to get your thoughts on this approach. (also see line 228 below). Thanks!